gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/explode/linker/ServiceLinker.java

```diff
@@ -64,6 +71,13 @@ public void link(Service service, Bundle bundle, Bundle targetBundle) {
                 targetBundle.getEntities(ServiceEnvironmentProperty.class).put(serviceEnvironmentProperty.getName(), serviceEnvironmentProperty);
             }
         }
+
+        if ("true".equals(portalManagedService) && ServiceAndPolicyLoaderUtil.isAnnotatePortalApisSet()) {
```

Comment:
What if this SERVICE entity is already annotated?

Reply:
@graju256 , annotations are added after yml files are generated, why do we expect annotations when exporting from gateway ?